src/store/modules/sitestatus/index.js

```diff
@@ -94,22 +101,22 @@ const getters = {
           text: 'operational',
           colorClass: 'is-green'
         }
-      // enclosure and weather both stale
+        // enclosure and weather both stale
       } else if (!enclosure_not_stale && !weather_not_stale) {
         return {
           text: 'offline',
           colorClass: 'is-grey'
         }
-      // enclosure is stale
+        // enclosure is stale
       } else if (!enclosure_not_stale && weather_not_stale) {
         return {
           text: 'enclosure not reporting',
           colorClass: 'is-yellow'
         }
-      // weather is stale
-      } else if (!enclosure_not_stale && weather_not_stale) {
+        // weather is stale
```

Comment:
These formatting changes seem unrelated to the feature, are they needed for this?